### ISSUE TITLE
Question regarding recipients.hash and config files in mounts

### ISSUE THREAD

**Issue by danielcb** [gopasspw/gopass#2530]
Hi,
We're using gopass (1.15.3). We're using a lot of different mounts (in git; one for each project) and just noticed that for every interaction with a mount (at least for adding/removing recipients or running gopass fsck) a config file inside the mount is created, but not added to git.

For clarification:

`~/.config/gopass/config`:
```
[...]

[mounts "some-mount"]
  path = /home/user/.password-store/some-mount

[mounts "other-mount"]
  path = /home/user/.password-store/other-mount
```

After each interaction with one of the mounts, there pops up a config (e.g. `.password-store/some-mount/config`) with the following content:

```
[recipients]
	hash = 50ec.......
```

The file is not added and gopass complains about it:

`⚠ Found untracked files: [config]`

Questions:
- I can't find any documentation about the recipients hash. What does it do? Is it one for each mount or one for all mounts?
- Why is this file created? If it is no bug, should this file be part of the mount? If so, why isn't it added and pushed by gopass?
- Why is the config file created inside the mount? Do these configs overwrite values inside the main config?

**Comment by dominikschulz** [comment]
* We need to document what it does.
* If it's supposed to be created in the mounts, it must be added to git. But see the next point first.
* I think it might be a bug that it's created inside the mounts. That doesn't make a lot of sense.

Will add more details once I get a chance to work on it.

**Comment by dominikschulz** [comment]
The main concern should have been addressed by #2535.

But I'd still like to:

- Check why configs in substores are not versioned and
- Document the recipients.hash feature properly
